lerobot/configs/default.yaml

```diff
@@ -57,3 +57,29 @@ wandb:
   disable_artifact: false
   project: lerobot
   notes: ""
+
+image_transform:
+  enable: true
```

Comment:
Should this default to true or false? @Cadene @marinabar

Reply:
I've set it to false by default